plugin/src/main/cpp/extensions/openxr_fb_spatial_entity_extension_wrapper.cpp

```diff
@@ -166,8 +166,13 @@ bool OpenXRFbSpatialEntityExtensionWrapper::create_spatial_anchor(const Transfor
 	Quaternion quat = Quaternion(p_transform.basis);
 	Vector3 pos = p_transform.origin;
 	XrPosef pose = {
-		{ quat.x, quat.y, quat.z, quat.w }, // orientation
-		{ pos.x, pos.y, pos.z }, // position
+		{ static_cast<float>(quat.x),
+				static_cast<float>(quat.y),
+				static_cast<float>(quat.z),
+				static_cast<float>(quat.w) },
+		{ static_cast<float>(pos.x),
+				static_cast<float>(pos.y),
+				static_cast<float>(pos.z) }
```

Comment:
```suggestion
		{ static_cast<float>(quat.x),
				static_cast<float>(quat.y),
				static_cast<float>(quat.z),
				static_cast<float>(quat.w) }, // orientation
		{ static_cast<float>(pos.x),
				static_cast<float>(pos.y),
				static_cast<float>(pos.z) } // position
```

Nit, but we've agreed on the style of adding the names of parameters in comments when initializing structs like this, so let's keep the comments. :)

Reply:
let me revise